src/main/java/com/qunite/api/web/controller/QueueController.java

```diff
@@ -70,9 +73,37 @@ public ResponseEntity<Integer> membersAmount(@PathVariable Long id) {
       @ApiResponse(responseCode = "200"),
       @ApiResponse(responseCode = "404", content = @Content())
   })
-  public ResponseEntity<Integer> memberPosition(@PathVariable(value = "id") Long queueId,
+  public ResponseEntity<Integer> memberPosition(@PathVariable Long id,
                                                 @PathVariable(value = "member-id") Long memberId) {
-    return ResponseEntity.of(queueService.getMemberPositionInQueue(memberId, queueId));
+    return ResponseEntity.of(queueService.getMemberPositionInQueue(memberId, id));
+  }
+
+  @PostMapping("/{id}/members/{member-id}")
+  @Operation(summary = "Enroll member to queue")
+  public ResponseEntity<Void> enrollMemberToQueue(@PathVariable Long id,
+                                                  @PathVariable(value = "member-id")
+                                                  Long memberId) {
+    queueService.enrollMemberToQueue(memberId, id);
+    return ResponseEntity.ok().build();
+  }
```

Comment:
It looks like the admin's endpoint to enroll any user to any queue. I believe we need another similar endpoint for user to enroll hisself.